lollypop: update to 1.4.26

sug0 opened the pull request:
<!-- Uncomment relevant sections and delete options which are not applicable -->

#### Testing the changes
- I tested the changes in this PR: **YES**

<!--
#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**|**NO**
-->

<!-- Note: If the build is likely to take more than 2 hours, please [skip CI](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#continuous-integration)
and test at least one native build and, if supported, at least one cross build.
Ignore this section if this PR is not skipping CI.
-->

#### Local build testing
- I built this PR locally for my native architecture
  - `Void 5.10.87-xanmod1_1 x86_64 AuthenticAMD notuptodate hold rrrmFFFFFFFF`

paper42 reviewed (commented):
I think we can also remove python3-youtube-dl and python3-dbus from depends.

sug0 commented:
Builds fine without `cmake` and `git`. Removed those. Also fixed up the `distfiles`.

> I think we can also remove python3-youtube-dl and python3-dbus from depends.

I tested the package without `python3-dbus` and it seems to be working fine.  There is no actual code using dbus I think. This is the output of `rg dbus` in the lollypop source tree:

```
search-provider/meson.build
1:service_dir = join_paths(get_option('datadir'), 'dbus-1', 'services')

search-provider/lollypop-sp.in
116:    'http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd'>

lollypop/mpris.py
84:    "http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
```
So, `python3-dbus` is gone as well. On the other hand, we still require `youtube-dl`. It's used as a dependency.
